.gitignore

```diff
@@ -71,3 +71,6 @@ typings/
 
 # Serverless directories
 .serverless
+
+# emacs save-files
+*~
```

Comment:
Probably a final newline will look nice for diffs :)

Reply:
haha, added :)